lib/studies/reply/adaptive_nav.dart

```diff
@@ -747,7 +748,12 @@ class _BottomAppBarActionItems extends StatelessWidget {
                         IconButton(
                           icon: const Icon(Icons.search),
```

Comment:
Ah I think because the Row is using a UniqueKey, it swaps it out when you navigate back. See if you can get around this to avoid the animation when the contents are the same

Reply:
That sounds right, i'll check it out thanks!

Reply:
Fixed by not setting a `key` for that `Row`. The animation still works because the other widgets have different keys.